lib/gollum/public/gollum/javascript/gollum.js.erb

```diff
@@ -164,10 +164,7 @@ function enable_code_blocks() {
     button.classList.add(
       "btn", 
       "btn-octicon", 
-      "border", 
```

Comment:
The copy to clipboard button was broken after updating primer. It looks like the `top-2` and `right-2` classes don't exist in the latest release of primer. For that reason, I'm now styling this button in `app.scss`.

Reply:
I'm not sure that diagnosis is correct. The screenshots below are from `master` with primer 20.8.

![Screenshot 2023-01-15 at 14 51 33](https://user-images.githubusercontent.com/571173/212544745-02a5e748-8375-4655-9677-f29fdc6455df.png)
![Screenshot 2023-01-15 at 14 51 49](https://user-images.githubusercontent.com/571173/212544749-6d93c93c-af96-4871-ab1c-6ca5b3289f54.png)

Reply:
Thanks for pointing this out. I just looked into it and it turns our `.right-2` is only defined in the primer `marketing` component:

```
$ grep -r "\.right-2" node_modules/@primer/css | awk -F":" '{print $1}'
node_modules/@primer/css/dist/primer.css
node_modules/@primer/css/dist/marketing-utilities.css
node_modules/@primer/css/dist/marketing.css
node_modules/@primer/css/dist/stats/primer.json
node_modules/@primer/css/dist/stats/marketing.json
node_modules/@primer/css/dist/stats/marketing-utilities.json
```

I am not sure whether it's worth importing the entire marketing style just for the `right-2` styling. :)